lib/displayStateMap.ts

```diff
@@ -2,7 +2,7 @@ import { NextFunction, Request, Response } from "express";
 import { IdentityContext } from "@aserto/node-authorizer/src/gen/cjs/aserto/authorizer/v2/api/identity_context_pb";
 
 import { Authorizer } from "./authorizer";
-import ResourceParamsMapper from "./authorizer/mapper/resource/params";
+import BodyResourceMapper from "./authorizer/mapper/resource/body";
```

Comment:
should we make the names consistent?

ResourceParamsMapper and ResourceBodyMapper?

Reply:
Actually since the rest of the names are things like `SubIdentityMapper`, `JWTIdentityMapper`, etc, I think it would be more consistent to have `BodyResourceMapper` (as you suggested) and then rename `ResourceParamsMapper` to `ParamsResourceMapper`. I will amend the PR.